src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/carts.yml

```diff
@@ -1,3 +1,51 @@
+admin_carts_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Order\CartController::indexAction
+    _legacy_controller: AdminCarts
+    _legacy_link: AdminCarts
+    _legacy_feature_flag: carts
```

Comment:
It's good to add the feature flag on the new routes, but you probably should also add it on the existing ones in this file